File: Exercism Codes/Python/raindrops.py
def convert(number):
    pl = ""
    if number % 3 == 0:
        pl += "Pling"
    if number % 5 == 0:
        pl += "Plang"
    if number % 7 == 0:
        pl += "Plong"
    return pl if len(pl) != 0 else str(number)

File: Exercism Codes/C++/Raindrops/raindrops.cpp
#include "raindrops.h"
#include <string>

namespace raindrops {
	std::string convert(int number) {
		std::string lst;
		if (number % 3 == 0) {
			lst += "Pling";
		}
		if (number % 5 == 0) {
			lst += "Plang";
		}
		if (number % 7 == 0) {
			lst += "Plong";
		}
		return ((lst.length() > 0) ? lst : std::to_string(number));
	}
}
